src/routes/console/organization-[organization]/deleteMember.svelte

```diff
@@ -25,6 +29,11 @@
             } else {
                 dispatch('deleted');
             }
+            invalidate(Dependencies.ACCOUNT);
```

Comment:
Why do we need to invalidate account?

Reply:
Not sure but it was there and I'm afraid to remove it :rofl: